Library/Homebrew/cmd/test-bot.rb

```diff
@@ -437,10 +438,35 @@ def formula(formula_name)
         formula_name
       end
 
-      test "brew", "uses", canonical_formula_name
-
       formula = Formulary.factory(canonical_formula_name)
 
+      if ARGV.include? "--fast"
+        formula_fetch_options = []
+        formula_fetch_options << "--force" if ARGV.include? "--cleanup"
+        formula_fetch_options << canonical_formula_name
+        test "brew", "fetch", "--retry", "--build-bottle",
+             *formula_fetch_options
+
+        if formula.bottled?
+          test "brew", "fetch", "--retry", *formula_fetch_options
+          install_args = ["--verbose", canonical_formula_name]
```

Comment:
Nope! It was used twice in an earlier iteration. I'll fix when I pull (to avoid another build)